packages/core/src/dispatcher.ts

```diff
@@ -128,23 +121,49 @@ export class Dispatcher {
    */
   async retrieveFromIPFS(cid: CID | string, path?: string): Promise<any> {
     try {
-      const asCid = typeof cid === 'string' ? new CID(cid) : cid
-
-      // Lookup CID in cache before looking it up IPFS
-      const cidAndPath = path ? asCid.toString() + path : asCid.toString()
-      const cachedDagNode = await this.dagNodeCache.get(cidAndPath)
-      if (cachedDagNode) return cloneDeep(cachedDagNode)
-
-      // Now lookup CID in IPFS and also store it in the cache
-      const dagNode = await this._ipfs.dag.get(asCid, { timeout: IPFS_GET_TIMEOUT, path })
-      await this.dagNodeCache.set(cidAndPath, dagNode.value)
-      return cloneDeep(dagNode.value)
+      return this._getFromIpfs(cid, path)
```

Comment:
Is there a reason we don't call `_restrictCommitSize()` here? If we should, then perhaps it can be called from inside `_getFromIpfs()`.

Reply:
Because this isn't used for loading commits.  That is the primary difference between `retrieveFromIPFS` and `retrieveCommit`.  The latter is specifically used to load Ceramic commits and restricts the commit size accordingly, while `retrieveFromIPFS` is for loading other random data from IPFS (primarily used for loading the anchor merkle tree currently).

Reply:
Ah gotcha.